include/GafferScene/EditScopeAlgo.h

```diff
@@ -123,6 +123,22 @@ GAFFERSCENE_API Gaffer::TweakPlug *acquireAttributeEdit( Gaffer::EditScope *scop
 GAFFERSCENE_API void removeAttributeEdit( Gaffer::EditScope *scope, const ScenePlug::ScenePath &path, const std::string &attribute );
 GAFFERSCENE_API const Gaffer::GraphComponent *attributeEditReadOnlyReason( const Gaffer::EditScope *scope, const ScenePlug::ScenePath &path, const std::string &attribute );
 
+// Sets
+// ==========
+//
+// These methods edit set membership for one or more locations.
+
+enum SetMembership
+{
+	IN,
+	OUT,
+	INHERIT
```

Comment:
Not sure what spirit in me decided on ALL CAPS, but agreed that it should be exorcised. I like `Inlcuded`, `Excluded` and `Unchanged` too. Another idea for `Unchanged` is `Passthrough`, though then I start thinking `Passthrough`? `PassThrough`?

Let's go with `Unchanged`.

Reply:
Fixed in 2668ddb9bbc7fe4c3b286735b215c77bc45111dd

Reply:
If you're willing to indulge me in some dithering, I'm now thinking `Added` and `Removed` would be better names than `Included` and `Excluded`, because they're a direct match for the terminology on the Set node itself. What do you think? Since I spent all my change-request tokens on the previous rename, I'm happy to make that change myself in a followup PR to save you the bother...

Reply:
Matching the Set node does seem like an improvement. I have a BOGO going on enum renaming right now, so it's no problem to take care of that. Renamed and squashed into 83f348a5d5dbd06b4cf0cea7a664918e93ffdad0!